app/services/plans/update_service.rb

```diff
@@ -97,6 +98,24 @@ def charge_model(params)
       model
     end
 
+    def process_minimum_commitment(plan, params)
+      minimum_commitment = plan.minimum_commitment || Commitment.new(plan:, commitment_type: 'minimum_commitment')
+
+      minimum_commitment.amount_cents = params[:amount_cents]
+      minimum_commitment.invoice_display_name = params[:invoice_display_name]
+      minimum_commitment.save!
```

Comment:
I think the logic here is not valid. If Commitment already exists and `invoice_display_name`  is already present, but if it is not sent in input parameters, it will be set to nil here. In update service (update mode) we should not set param that is not passed in params